packages/algob/src/internal/core/config/config-validation.ts

```diff
@@ -156,7 +160,12 @@ export function getValidationErrors(config: any): CfgErrors {
 				errors.push(net, "host", host, "hostname string (eg: http://example.com)");
 			}
 			const token = hcfg.token;
-			if (typeof token === "string" && token.length < 10) {
+			if (
+				typeof token === "string" &&
+				token.length < 10 &&
+				host !== testnetURL &&
+				host !== mainnetURL
+			) {
 				errors.push(net, "token", token, "string (with length > 10)");
```

Comment:
This doesn't look right either, You're validating upon host values, but pushing `"token", token ..` in `errors`.